recipe/build.sh

```diff
@@ -1,6 +1,12 @@
 set -e
 
-./configure --prefix=$PREFIX --enable-official-khronos-headers
+./bootstrap
+
+./configure \
+  --prefix=$PREFIX \
+  --enable-official-khronos-headers \
+  --enable-custom-vendordir=$PREFIX/etc/OpenCL/vendors
```

Comment:
Should we create this directory as well? It seems when I install the package the directory isn't present. That said, `conda-build` has not been great historically about preserving empty directories. So it may need something like an empty `.keep` file or similar.